app/logic/Mail/EWS/EWSAccount.ts

```diff
@@ -89,6 +93,8 @@ export class EWSAccount extends MailAccount {
     await this.oAuth2?.logout();
   }
 
+  // This uses Exchange to construct the message rather than
+  // building the MIME ourselves.
```

Comment:
When the comment pertains to an entire function, like this one here, and several other comments below, can you please put it in JSDoc format? That allows the IDE to pick up the comment and show it even when hovering over the caller (!).

Like so:
```suggestion
  /** This uses Exchange to construct the message rather than
   * building the MIME ourselves. */
```